mitodl/ocw-hugo-themes
# feat: compact footer

#### Pre-Flight checklist


#### What are the relevant tickets?
closes https://github.com/mitodl/ocw-hugo-themes/issues/861

#### What's this PR do?
- Introduces a new compact footer (aka footer-v2)
- Currently, footer-v2 files have been created in `base-theme` so if we have to use them in other themes (www, course-offline), we can use them without creating them again.

#### How should this be manually tested?
- Checkout this branch OR visit [netlify deployed version](https://ocw-hugo-themes-course-v2-pr-868--ocw-next.netlify.app/)
- Verify that the footer in CHP and other interior pages is according to the [design](https://projects.invisionapp.com/d/main?origin=v7#/console/21537330/469768935/preview?scrollOffset=0)
- Verify that all the links/buttons on the footer are working and redirecting to the correct locations.
- Verify that all the content/text on footer is correct (grammatically too) 
- Verify that footer is completely responsive on all screen sizes.
- Verify that footer is working fine on all browsers.

#### Screenshots (if appropriate)

![screencapture-localhost-3000-2022-09-28-16_54_25](https://user-images.githubusercontent.com/93309234/192772598-4ff3f2d3-59fa-4518-a642-461095187d92.png)

<img width="921" alt="image" src="https://user-images.githubusercontent.com/93309234/194318779-d804af63-7240-407c-81d0-2c9747591704.png">

<img width="488" alt="image" src="https://user-images.githubusercontent.com/93309234/194318859-0b2f4899-3b66-43c4-a019-9353bead37fc.png">
